File: app/src/main/java/com/example/simonastojanovic/memeapp/domain/FetchMemeUseCase.java
package com.example.simonastojanovic.memeapp.domain;

import android.arch.lifecycle.MutableLiveData;

import com.example.simonastojanovic.memeapp.model.MemesItem;

import java.util.ArrayList;

public interface FetchMemeUseCase {

    void execute(MutableLiveData<ArrayList<MemesItem>> memeResponseLiveData);

}

File: app/src/main/java/com/example/simonastojanovic/memeapp/domain/FetchMemeUseCase.kt
package com.example.simonastojanovic.memeapp.domain

import android.arch.lifecycle.MutableLiveData
import com.example.simonastojanovic.memeapp.model.MemesItem
import java.util.*

interface FetchMemeUseCase {

    fun execute(memeResponseLiveData: MutableLiveData<ArrayList<MemesItem>>)

}